lib/xenops_server.ml

```diff
@@ -99,6 +99,8 @@ type atomic =
 	| VM_save of (Vm.id * flag list * data)
 	| VM_restore of (Vm.id * data)
 	| VM_delay of (Vm.id * float) (** used to suppress fast reboot loops *)
+	| VGPU_save of (Vgpu.id * data)
```

Comment:
Let's change this to `VM_save_vgpu of (Vm.id * Vgpu.id * data)` and equivalent for restore.